<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="POSTACI" failures="2" tests="7" name="TestNgProject" time="0.459" errors="0" timestamp="2025-04-14T13:15:59 TRT">
  <testcase classname="ondia.tests.C25ListenersTest" name="test01" time="0.056"/>
  <testcase classname="ondia.tests.C25ListenersTest" name="test02" time="0.004">
    <skipped/>
  </testcase> <!-- test02 -->
  <testcase classname="ondia.tests.C25ListenersTest" name="test02" time="0.003">
    <skipped/>
  </testcase> <!-- test02 -->
  <testcase classname="ondia.tests.C25ListenersTest" name="test02" time="0.004">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
at ondia.tests.C25ListenersTest.test02(C25ListenersTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- test02 -->
  <testcase classname="ondia.tests.C25ListenersTest" name="test03" time="0.004">
    <skipped/>
  </testcase> <!-- test03 -->
  <testcase classname="ondia.tests.C25ListenersTest" name="test04" time="0.003"/>
  <testcase classname="ondia.tests.C25ListenersTest" name="test05" time="0.002">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at ondia.tests.C25ListenersTest.test05(C25ListenersTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- test05 -->
</testsuite> <!-- TestNgProject -->
